src/dbnode/storage/index/block_test.go

```diff
@@ -1933,6 +1947,12 @@ func TestBlockAggregate(t *testing.T) {
 	spans := mtr.FinishedSpans()
 	require.Len(t, spans, 2)
 	require.Equal(t, tracepoint.BlockAggregate, spans[0].OperationName)
+
+	for _, v := range scope.Snapshot().Counters() {
```

Comment:
can use `tallytest.AssertCounterValue`

Reply:
:+1: nice, did not know about this one

Reply:
Ah actually it requires a full set of tags for the checker, not super relevant for this test